docs/eos-learn/system-resources.md

```diff
@@ -5,38 +5,44 @@ sidebar_label: System Resources
 ---
 
 ## Type of Resources
+
 The accounts will be able to use the following resources in the network:
 
-## RAM
+### RAM
+
 In an EOSIO network, RAM is the memory storage space where the blockchain stores data, it is measured in kilobytes (KiB). If your contract needs to store data in a blockchain table, like in a database, then it can store it in the blockchain's RAM.
 
 RAM is a very important resource and it is a limited one. It is used when executing many actions that are available on the blockchain, when creating a new account for example the information for that account is stored in the blockchain’s memory. Also when an account accepts a new type of token a new record has to be created somewhere in the blockchain memory that holds the balance of the new token accepted, and that memory, the storage space on the blockchain, has to be purchased either by the account that transfers the token or by the account that accepts the new token type.
 
 RAM is referred to as `memory` in the following `cleos get account` output:
 
-	memory:
-	quota: 86.68 KiB
-	used:   11.62 KiB
+```C++
+memory:
+quota: 86.68 KiB
+used: 11.62 KiB  
+```
 
-You can find more details about RAM as a resource [here](https://developers.eos.io/manuals/eosio.contracts/latest/key-concepts/ram)
+You can find more details about RAM as a resource [here](https://developers.eos.io/manuals/eosio.contracts/latest/key-concepts/ram).
+
+### CPU
 
-## CPU
 CPU is processing power, the amount of CPU an account has is measured in microseconds (μs), it is referred to as `CPU bandwidth` the `cleos get account` command outputs the amount of processing time an account has at its disposal when pushing actions to a contract.
 
 You can find more details about the CPU as a resource [here](https://developers.eos.io/manuals/eosio.contracts/latest/key-concepts/cpu)
 
-## Network (NET)
+### Network (NET)
+
 As CPU and RAM, NET is also a very important resource in EOSIO-based blockchains. NET is the network bandwidth measured in bytes of transactions and it is referred to as `net bandwidth`.
 
-You can find more details about NET as a resource [here](https://developers.eos.io/manuals/eosio.contracts/latest/key-concepts/net)
+You can find more details about NET as a resource [here](https://developers.eos.io/manuals/eosio.contracts/latest/key-concepts/net).
 
-## Account resource limits
+## Account Resource imits
```

Comment:
"Limits"

Reply:
I think it was extra finger, good observation.